scientific_library/tvb/simulator/simulator.py

```diff
@@ -50,16 +50,24 @@
 from .history import SparseHistory
 from tvb.basic.neotraits.api import HasTraits, Attr, NArray, List, Float
 
+from tvb.simulator.models.reduced_wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
+from tvb_multiscale.config import CONFIGURED
+
 
 # TODO with refactor, this becomes more of a builder, since iterator will account for
 # most of the runtime associated with a simulation.
 class Simulator(HasTraits):
+
+    tvb_spikeNet_interface = None
+    configure_spiking_simulator = None
+    run_spiking_simulator = None
+
     """A Simulator assembles components required to perform simulations."""
 
     connectivity = Attr(
         field_type=connectivity.Connectivity,
         label="Long-range connectivity",
-        default=None,
+        default=CONFIGURED.DEFAULT_SUBJECT["connectivity"],
```

Comment:
This could be left to None, and then perhaps we can easier remove the dependency towards  CONFIGURED

Reply:
Done